package.json

```diff
@@ -1,19 +1,32 @@
 {
   "name": "liquidity-provision-scripts",
-  "version": "0.1.0",
-  "description": "",
+  "description": "Scripts to deploy and manage automated traders on the Gnosis Protocol exchange",
+  "private": true,
+  "keywords": [
+    "Gnosis Protocol",
+    "liquidity",
+    "trading"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "github:https://github.com/gnosis/dex-liquidity-provision"
+  },
+  "bugs": {
+    "url": "https://github.com/gnosis/dex-liquidity-provision/issues"
+  },
   "devDependencies": {
     "@gnosis.pm/mock-contract": "^3.0.0",
     "eslint": "^6.8.0",
     "eslint-plugin-no-only-tests": "^2.4.0",
     "eslint-plugin-react": "^7.19.0",
-    "eth-lightwallet": "^4.0.0",
-    "ethereumjs-abi": "^0.6.5",
     "ganache-cli": "6.9.1",
     "prettier": "^2.0.5",
-    "run-with-testrpc": "^0.3.0",
+    "sol-merger": "^2.0.1",
```

Comment:
when was this introduced and what for?

Reply:
It's used to produce an "Etherscan-friendly" version of a contract's source code by collecting all contracts involved in the compilation of a contract into a single file. It's only required for running `yarn flatten`.